lib/thor.rb

```diff
@@ -439,6 +439,17 @@ def disable_required_check?(command) #:nodoc:
       command && disable_required_check.include?(command.name.to_sym)
     end
 
+    # Checks if a specified command exists.
+    #
+    # ==== Parameters
+    # command_name<String>:: The name of the command to check for existence.
+    #
+    # ==== Returns
+    # Boolean:: +true+ if the command exists, +false+ otherwise.
+    def command_exists?(command_name)
```

Comment:
Do we need this method in the public API? Should we `:nodoc:` it instead? I don't think we have a need to expose this to users.

Reply:
Thank you for reviewing. I added `:nodoc:`.